Documentation/Books/Manual/Programs/Arangorestore/Examples.md

```diff
@@ -115,6 +115,15 @@ collections being processed before all [edge collection](../../Appendix/Glossary
 data into edge collections will have the document collections linked in edges (*_from* and
 *_to* attributes) loaded.
 
+To restrict reloading to specific views, there is is the *--view* option.
```

Comment:
Simran, probably we want to consider a "Restoring Views" or similar subsection, so it is easily visible, wdyt?and write that views are restored by default if they are included in the backup but the option --view can be used to restrict dump only to some specific views?

we will also need to create section about cluster, e.g. something like "Cluster Restore" or "Restoring into a Cluster", in a similar way we did with the dump, and use the keyword Cluster in the headline of such section instead of sharing (more clear) - but this is offtopic here and a different task